test/failure_test.py

```diff
@@ -483,5 +483,34 @@ def testVersionMismatch(self):
         ray.__version__ = ray_version
 
 
+class WarningTest(unittest.TestCase):
+    def tearDown(self):
+        ray.worker.cleanup()
+
+    def testExportLargeObjects(self):
+        import ray.ray_constants as ray_constants
+
+        ray.init(num_workers=1)
+
+        large_object = np.zeros(2 * ray_constants.PICKLE_OBJECT_WARNING_SIZE)
+
+        @ray.remote
+        def f():
+            large_object
+
+        # Make sure that a warning is generated.
+        wait_for_errors(b"pickling_large_object", 1)
```

Comment:
Does `wait_for_errors` read warnings?

Reply:
Yeah, basically, it reads everything pushed to the `error_info` queue